Maquetado/scss/_colors.scss

```diff
@@ -1,2 +1,3 @@
 $black: #000;
 $mediumGray: #828282;
+$woloxGreen: #b5d34d
```

Comment:
This is ok, but is reommendo you to use this app to choose the color name: https://chir.ag/projects/name-that-color/#6195ED